resources/conditions.js

```diff
@@ -3,23 +3,30 @@
 // For consistency with Responses, Conditions
 // are also functions.
 
-let Conditions = {
-  targetIsYou: () => (data, matches) => {
-    return data.me == matches.target;
-  },
-  targetIsNotYou: () => (data, matches) => {
-    return data.me !== matches.target;
-  },
-  caresAboutAOE: () => (data) => {
-    return data.role == 'tank' || data.role == 'healer' || data.CanAddle() || data.job == 'BLU';
-  },
-  caresAboutMagical: () => (data) => {
-    return data.role == 'tank' || data.role == 'healer' || data.CanAddle() || data.job == 'BLU';
-  },
-  caresAboutPhysical: () => (data) => {
-    return data.role == 'tank' || data.role == 'healer' || data.CanFeint() || data.job == 'BLU';
-  },
-};
-
-// TODO: Convert into static class
-export default Conditions;
+export default class Conditions {
+  static targetIsYou() {
+    return (data, matches) => {
+      return data.me == matches.target;
+    };
```

Comment:
Since these are arrow functions anyway how about using implicit returns?
```suggestion
    return (data, matches) => data.me == matches.target;
```

Reply:
Sure!